### ISSUE TITLE
allow users to decide how to merge numbers from multiple samples per patient

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#5049]
Currently we use average values. We use max for mutation_count. For some other attribute, max may also make sense, e.g. msi_score.

Can we give the user option to use different ways to merge numbers, e.g. ave, min, max, median? It should be a per-track setting so the track menu may be the right place to implement this.

**Comment by adamabeshouse** [comment]
what do you think about adding a menu to the toolbar, like "Track Settings", and from there we can change these settings? Rather than adding it to the oncoprint library track menu. That would make it easier to save the oncoprint state into the session as we want to, and it will be much easier and more flexible to work with if we want to modify things now and in the future.

**Comment by adamabeshouse** [comment]
@jjgao yes I was thinking we could list the tracks in the menu, and have their settings expand on rollover/click or something like that
